### PR TITLE
made some improvements to index.md

### PR DESCRIPTION
**Pull request by emelie042**
please merge

### CODE REVIEW

File: index.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
-# Markus Wallin
+# Emelie Eriksson
```
Comment (MarkusWallin388) [inline index.md]:
```suggestion
# Markus Wallin
```

File: index.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
...
 Collaborative software engineer
 ## My projects
 * Collabortive Software Project in Computational Physics
 ## My skills
-* Git and GitHub
\ No newline at end of file
+* Git and GitHub, being awesome
```
Comment (MarkusWallin388) [inline index.md:6]:
```suggestion
* Git and GitHub, being awesome and being better than alice and emelie
```

**Review by MarkusWallin388 — CHANGES_REQUESTED** [review]
This was a good pull request but I have even better changes

File: index.md (hunk 1 above)
Comment (emelie042) [reply index.md]: nej

**Pull request closed without merging**